### PR TITLE
🤵🏽‍♀️ Fix conflict of api.js

### PR DESCRIPTION
**Pull request by SolMon9099**
## Description :star2:

Fix conflict of api.js

### CODE REVIEW

File: frontend/src/utils/api.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ export const fetchAccountById = (id, type) => {
 export const fetchAccounts = (type, restricted = null) => {
   const requestExtension =
     `/accounts/${type}` + (restricted === null ? "" : "?restricted=True");
-  return instance.get(requestExtension).then(
```
Comment (Leundai) [inline frontend/src/utils/api.js:56]: Don't hardcode these query params instead add them like `fetchAccountById`
[Link to code](https://github.com/hack4impact-uiuc/mentee/blob/3a203777d56fb6f6e99844d5b3ce9f1664d7a56c/frontend/src/utils/api.js#L38-L51)

**Review by Leundai — CHANGES_REQUESTED** [review]

File: frontend/src/components/pages/Gallery.js [hunk 2]
```diff
@@ -13,7 +12,8 @@ import {
   fetchMentors,
   getDisplaySpecializations,
   getDisplayLanguages,
-} from "utils/api";
+  fetchPartners,
+} from "../../utils/api";
```
[the remark this answers is not in the document]
Comment (SolMon9099) [reply frontend/src/components/pages/Gallery.js]: I have fixed it

File: frontend/src/components/pages/MenteeGallery.js [hunk 3]
```diff
@@ -1,5 +1,10 @@
 import React, { useState, useEffect } from "react";
-import { fetchPartners, fetchMentees } from "../../utils/api";
+import {
+  fetchPartners,
+  fetchMentees,
+  getDisplaySpecializations,
+  getDisplayLanguages,
+} from "../../utils/api";
```
[the remark this answers is not in the document]
Comment (SolMon9099) [reply frontend/src/components/pages/MenteeGallery.js]: I have fixed it

File: frontend/src/utils/api.js (hunk 1 above)
Comment (SolMon9099) [reply frontend/src/utils/api.js]: I have fixed it

**Review by robert-dm — APPROVED** [review]

**Review by Leundai — APPROVED** [review]
lgtm

**Pull request merged**